library/alloc/src/vec/cow.rs

```diff
@@ -15,6 +15,19 @@ impl<'a, T: Clone> From<&'a [T]> for Cow<'a, [T]> {
     }
 }
 
+#[stable(feature = "cow_from_array", since = "1.8.0")]
```

Comment:
```suggestion
#[stable(feature = "cow_from_array", since = "CURRENT_RUSTC_VERSION")]
```

IIUC `since` fields are advisory only though, so 🤷.